Devscord.DiscordFramework/Commons/Exceptions/ChannelNotFoundException.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Devscord.DiscordFramework.Commons.Exceptions
+{
+    public class ChannelNotFoundException : BotException
+    {
+        public ChannelNotFoundException(string mention) : base(mention)
```

Comment:
powinna być przygotowana odpowiedź do tego wyjątku -> sprawdź jak to robimy w innych przypadkach